File: Ad-Hoc/URI 1893.pas
var
    inicio : int64;
    final : int64;

begin
    readLn(inicio, final);
	
	if ((inicio >= 0) and (final <= 2)) then
		writeln('nova')
	else if ((final > inicio) and (final <= 96)) then
		writeln('crescente')
	else if ((inicio >= final) and (final <= 96)) then
		writeln('minguante')
	else 
		writeln('cheia')
end.

File: Ad-Hoc/URI 1893.c
#include <stdio.h>
 
int main() { 
    int inicio, final;

    scanf("%d %d", &inicio, &final);
    if (inicio >= 0 && final <= 2) {
        printf("nova\n");
    } else if (final > inicio && final <= 96) {
        printf("crescente\n");
    } else if (inicio >= final && final <= 96) {
        printf("minguante\n");
    } else { 
        printf("cheia\n");
    }
    
    return 0;
}